pymc/distributions/multivariate.py

```diff
@@ -2077,6 +2078,9 @@ def rng_fn(cls, rng: np.random.RandomState, mu, W, alpha, tau, size):
         Journal of the Royal Statistical Society Series B, Royal Statistical Society,
         vol. 63(2), pages 325-338. DOI: 10.1111/1467-9868.00288
         """
+        if np.all(alpha >= 1) or np.all(alpha <= -1):
```

Comment:
I assume you mean this?

```suggestion
        if np.any(alpha >= 1) or np.any(alpha <= -1):
```